supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/statuspolling/JesApiQueryManager.scala

```diff
@@ -40,9 +48,13 @@ class JesApiQueryManager(val qps: Int Refined Positive) extends Actor with Actor
   resetWorker()
 
   override def receive = {
-    case DoPoll(run) => workQueue :+= JesStatusPollQuery(sender, run)
+    case DoPoll(run) => workQueue :+= makePollQuery(sender(), run)
```

Comment:
Though it can only be called in the context of a partial function from Any to Unit.  Maybe it's just a penance for that overarching sin.